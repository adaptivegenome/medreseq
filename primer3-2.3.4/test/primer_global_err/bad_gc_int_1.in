SEQUENCE_ID=bad_gc_int_1
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_MAX_GC=-2
=
