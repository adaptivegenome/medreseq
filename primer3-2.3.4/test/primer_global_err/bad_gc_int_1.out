SEQUENCE_ID=bad_gc_int_1
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_MAX_GC=-2
PRIMER_ERROR=Illegal value for PRIMER_INTERNAL_OLIGO_GC
=
